Automattic/crowdsignal-ui
# Add/url input variation

Building off the email block, this PR adds another variation of the text input form for a URL input.

To add a new block, add a variation to /block-editor/text-input/variations.js  Add a validator to blocks/text-input/validations.js (if required) and then add an icon to /icons

You then end up with something like this:

<img width="679" alt="Screen Shot 2022-03-25 at 10 54 37 AM" src="https://user-images.githubusercontent.com/12895386/160147983-c7817839-0e38-4637-ab0c-4cb096cfdf63.png">

Testing instructions:
Apply this PR and do the yarn thing then visit https://crowdsignal.localhost:9000/project and create a new project.

Click the block inserter and you should see the new URL form input block, add it and then preview your form.

Try adding an invalid URL and see if the error message appears

Correct the URL, verify the error goes away

Note, right now the validator doesn't look for http(s):// but if anyone has a strong feeling about that, the regex can be updated to require it.
